posts/2019/04/whats-new-in-sympy-14.md

```diff
@@ -0,0 +1,228 @@
+<!--
+.. title: What's New in SymPy 1.4
+.. slug: whats-new-in-sympy-14
+.. date: 2019-04-29
+.. author: Aaron Meurer
+.. tags: sympy, Labs, mathjax
```

Comment:
the post is unrelated to mathjax right? if so, I would remove the tag. Also, spell it SymPy?

Reply:
I'm unclear if the `mathjax` tag or `has_math` enables MathJax. Note that the `mathjax` tag is automatically hidden due to a setting in conf.py.

Reply:
I guess it still works if I remove the tag.